test/modules/fundingManager/rebasing/utils/mocks/ElasticReceiptBaseV1Mock.sol

```diff
@@ -4,32 +4,31 @@ pragma solidity ^0.8.0;
 import {ERC20} from "@oz/token/ERC20/ERC20.sol";
 
 import {
-    ElasticReceiptTokenUpgradeable_v1,
-    ElasticReceiptTokenBase_v1
-} from "@fm/rebasing/abstracts/ElasticReceiptTokenUpgradeable_v1.sol";
+    ElasticReceiptBase_v1,
+    IOrchestrator_v1
+} from "@fm/rebasing/abstracts/ElasticReceiptBase_v1.sol";
 
-contract ElasticReceiptTokenUpgradeableV1Mock is
-    ElasticReceiptTokenUpgradeable_v1
-{
+contract ElasticReceiptBaseV1Mock is ElasticReceiptBase_v1 {
     // The token's underlier.
     // Is of type ERC20.
     address public underlier;
```

Comment:
The variable is just in the mock, doesn't exist in the actual contract. And it was using the `constructor` which (as we deploy via a proxy because of `_ disableInitializer()` wasn't working anymore.